q2cli/tests/test_tools.py

```diff
@@ -247,6 +247,18 @@ def test_export_visualization_to_dir_success_message(self):
                   'directory %s\n' % (self.viz, output_path)
         self.assertEqual(success, result.output)
 
+    def test_export_to_filename_without_path(self):
+        current_dir = os.getcwd()
+        os.chdir(self.tempdir)
```

Comment:
Do you need to chdir? If so, please add a comment explaining why.

Reply:
I have an idea --- please split these new tests out into their own test class --- the chdir business can be handled in the `setUp` and `tearDown` methods, and the comment can be moved into those hooks.